.github/workflows/ci.yml

```
@@ -360,17 +358,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Authenticate to Google Cloud
-        uses: google-github-actions/auth@v2
-        with:
-          credentials_json: ${{ secrets.GCP_SA_KEY }}
-
       - name: Download zcash-btest artifact
-        uses: ./.github/actions/gcs-download-cloud-storage
+        uses: actions/download-artifact@v4
         with:
-          path-prefix: ${{ github.run_id }}/${{ matrix.name }}
-          source: ${{ format('src/test/test_bitcoin{0}', matrix.file_ext) }}
-          destination: ./
+          name: ${{ format('./src/test/test_bitcoin{0}', matrix.file_ext) }}-${{ matrix.name }}
+          path: ${{ format('./test_bitcoin{0}', matrix.file_ext) }}
```

```suggestion
          name: ${{ format('zcashd{0}', matrix.file_ext) }}-${{ matrix.name }}
          path: ./src
```
I think this is right.